### ISSUE TITLE
Lots of logs are being printed ("Component already has a Path Item Object") when I add endpoints to the client

### ISSUE THREAD

**Issue by amitmalav** [godaddy/kubernetes-client#452]
Hello,
I'm currently on `6.12.0` version.

I'm seeing lots of errors like this when I add `customResourceDefinition` endpoint to client
```
Component / already has a Path Item Object
Component / already has a Path Item Object
Component / already has a Path Item Object
Component / already has a Path Item Object
Component / already has a Path Item Object
Component / already has a Path Item Object
```
This specifically happens when I call `addCustomResourceDefinition ` method multiple times for the same crd (Printing 2 times per call)

I understand that I need not register same crd twice but is there any way such printing can be avoided even if I register same crd endpoint multiple times.

**How to reproduce:**
```js
const Client = require('kubernetes-client').Client
const config = require('kubernetes-client').config

const crd = require('./crontabs-crd.json')

async function main () {
  try {
    const client = new Client({ config: config.fromKubeconfig(), version: '1.9' })

    client.addCustomResourceDefinition(crd)
    client.addCustomResourceDefinition(crd)
    client.addCustomResourceDefinition(crd)
// This will print 4 such lines on console
  } catch (err) {
    console.error('Error: ', err)
  }
}

main()
```

**Comment by silasbw** [comment]
What's the reason why you're adding the same CRD multiple times?  We can definitely consider suppressing the log messages if they aren't helpful.

**Comment by amitmalav** [comment]
We have multiple modules using the same client. There is no easy way programmatically to avoid calling `addCustomResourceDefinition` for same crd. It'd have been helpful if there was some way to know that a particular crd has already been added which is not the case currently.
In the earlier version of the library that we were using (`5.3.0`), adding crd multiple times was not a problem. We started facing this issue since upgrading to `6.12.0`. It would certainly be helpful if these log messages are suppressed.

**Comment by silasbw** [comment]
OK, understood.  I'll address this somehow.

**Comment by silasbw** [comment]
Included the change/fix with kubernetes-client@7.0.1 :beers:

**Comment by amitmalav** [comment]
Awesome. I just tested with `7.0.1` and everything seems smooth now.
Thanks a ton @silasbw
